barretenberg/cpp/src/barretenberg/bb/cli.hpp

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+namespace bb {
```

Comment:
We split main.cpp into a thin call to cli.cpp so that we can use it the new 'main' calling bb_cli_bench